src/index.spec.ts

```diff
@@ -40,10 +41,10 @@ type Person = { name: string; likes: Person[] };
 const Person: Runtype<Person> = Lazy(() => Record({ name: String, likes: Array(Person) }));
 
 class SomeClass {
-  constructor(public n: number) {}
+  constructor(public n: number) { }
```

Comment:
because I'm kind of messing up 🤕

Reply:
Well it revealed that Prettier wasn't running correctly in CI, so that's good to know :)